modules/apps/staging/staging-taglib/src/main/resources/META-INF/resources/portlet_list/page.jsp

```diff
@@ -25,13 +31,7 @@
 
 		PortletDataHandler portletDataHandler = portlet.getPortletDataHandlerInstance();
 
-		if (!portletDataHandler.isEnabled(company.getCompanyId())) {
-			continue;
-		}
-
-		StagingGroupHelper stagingGroupHelper = StagingGroupHelperUtil.getStagingGroupHelper();
-
-		if (portletDataHandler.isCompany() != stagingGroupHelper.isCompanyGroup(group)) {
+		if (!portletDataHandler.isEnabled(company.getCompanyId()) && (portletDataHandler.isCompany() != companyGroup)) {
```

Comment:
Are you sure this condition is the same as before?

Reply:
Hey @ccorreagg!
Here what I did was moving the creation of `stagingGroupHelper` to an upper level. That is used to get `companyGroup`, as its used in two places, I created the variable for it somewhere accessible by both.
That leaves us with two if conditions that return the same thing, and thats what I merged into this line.

Reply:
Yes, but before this, you had an OR condition and now you have an AND condition.

What if the company is not enabled but it is a company group?

Before: The item would not be processed
After: The item is being processed

:warning:

Reply:
Absulutely! This should be an OR. Let me think of a way of testing it and I can add a test to the PR.